src/codecoverage/bot/code_coverage_bot/phabricator.py

```diff
@@ -118,12 +117,23 @@ def generate(self, report, changesets=None):
                         # This means the file has been removed by this changeset, and maybe was brought back by a following changeset.
                         continue
 
+                    # Calc lines added by this patch
+                    lines_added = sum([
+                        parse_revision_id(line['desc']) == revision_id
+                        for line in build_annotate
+                    ])
+
                     # Apply the coverage map on the annotate data of the changeset of interest.
-                    results[revision_id][path] = self._apply_coverage_map(annotate, coverage_map)
+                    coverage = self._apply_coverage_map(annotate, coverage_map)
+                    results[revision_id][path] = {
+                        'lines_added': lines_added,
+                        'lines_covered': sum(c == 'C' for c in coverage),
```

Comment:
We should only consider the lines added which were covered (we want to notify when a changeset introduces a large number of uncovered lines, so the message should say "X lines covered out of Y added")